vnext/ReactUWP/Views/FrameworkElementViewManager.cpp

```diff
@@ -21,18 +21,54 @@
 #include <winrt/Windows.UI.Xaml.Hosting.h>
 #include <winrt/Windows.UI.Xaml.h>
 
+#include "Utils/PropertyHandlerUtils.h"
+
 #include "DynamicAutomationProperties.h"
 
 namespace winrt {
 using namespace Windows::UI::Xaml;
 using namespace Windows::UI::Xaml::Controls;
 using namespace Windows::UI::Xaml::Automation;
 using namespace Windows::UI::Xaml::Automation::Peers;
+using namespace Windows::Foundation::Collections;
 } // namespace winrt
 
 namespace react {
 namespace uwp {
 
+template <>
+struct json_type_traits<winrt::react::uwp::AccessibilityAction> {
+  static winrt::react::uwp::AccessibilityAction parseJson(const folly::dynamic &json) {
+    auto action = winrt::react::uwp::AccessibilityAction();
+
+    for (auto &item : json.items()) {
+      if (item.first == "name") {
+        action.Name = react::uwp::asHstring(item.second);
+      } else if (item.first == "label") {
+        action.Label = react::uwp::asHstring(item.second);
+      }
+    }
+    return action;
+  }
+};
+
+template <>
+struct json_type_traits<winrt::IVector<winrt::react::uwp::AccessibilityAction>> {
+  static winrt::IVector<winrt::react::uwp::AccessibilityAction> parseJson(const folly::dynamic &json) {
```

Comment:
parseAccessibilityActions (or something :)) instead of parseJson.
dynamics are used a lot with json but they are not the same thing.
Naming this parseJson may confuse the next person who reads this.

Reply:
I will not make change.
parseJson is static function name of json_type_traits template.
And here json_type_traits<winrt::IVector<winrt::react::uwp::AccessibilityAction>> is just a template specialization.

---
In reply to: [339680046](https://github.com/microsoft/react-native-windows/pull/3475#discussion_r339680046) [](ancestors = 339680046)